features/importing/archivist.feature

```diff
@@ -78,10 +78,9 @@ Feature: Archivist bulk imports
   Then I should see import confirmation
     And I should see "Name change"
   When the system processes jobs
+  # Importer assumes dreamwidth email for works from there
   Then 1 email should be delivered to "jennyst@dreamwidth.org"
```

Comment:
The importer code automatically constructs from username and domain, so it needs to match that to test it effectively. If we want to avoid using a person, we could use a community instead, but I'm not sure that really helps much in terms of privacy.